src/js/index.js

```diff
@@ -258,6 +262,10 @@ const initPlugin = function(player, options) {
     options.errors = videojs.mergeOptions(options.errors, errors);
   };
 
+  reInitPlugin.disableProgress = function(disabled) {
```

Comment:
Should we call `onPlayStartMonitor` in here? That'll cleanup the current monitors, specifically, the progress monitor, and then re-adds the listeners.